package.json

```diff
@@ -22,7 +22,7 @@
   },
   "license": "MIT",
   "author": "stylelint",
-  "main": "lib/index.js",
+  "main": "lib/index.cjs",
```

Comment:
I am not sure, but I think the agnostic `"lib/index"` should work.
Can you try it out?

Reply:
[It seems that automatic extension resolution has been removed.](https://stackoverflow.com/posts/46900211/revisions)
You can disregard my suggestion.